test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java

```diff
@@ -514,6 +516,10 @@ public Collection<Class<? extends Plugin>> getPlugins() {
         return plugins;
     }
 
+    public Collection<PluginInfo> getPluginInfos() {
```

Comment:
Where this method is used? (outside of `InternalTestCluster`) ?

Reply:
Only within here, I removed the method and called `nodeConfigurationSource.additionalNodePlugins()` inline.